### PR TITLE
Added basic bool properties file

### PR DESCRIPTION
**Pull request by C0dingWithFun**
Added visitBoolProperties and visitBoolMethods in core_types

### CODE REVIEW

File: lib/runtime/runtime.dart [hunk 1]
```diff
@@ -9,6 +9,9 @@ import 'package:Birb/parser/data_type.dart';
 import 'package:Birb/runtime/standards.dart';
 import 'package:Birb/lexer/token.dart';
 
+import '../core_types/core_types.dart';
```
Comment (Andy-Python-Programmer) [inline lib/runtime/runtime.dart:12]: Use package imports

File: lib/runtime/runtime.dart [hunk 2]
```diff
@@ -9,6 +9,9 @@ import 'package:Birb/parser/data_type.dart';
...
+import '../utils/ast/ast_node.dart';
```
Comment (Andy-Python-Programmer) [inline lib/runtime/runtime.dart:13]: Again use package imports a preferred!

File: lib/core_types/bool.dart [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import 'package:Birb/runtime/runtime.dart';
+import 'package:Birb/utils/ast/ast_node.dart';
+import 'package:Birb/utils/ast/ast_types.dart';
+import 'package:Birb/utils/exceptions.dart';
+
+import '../utils/ast/ast_node.dart';
```
Comment (Andy-Python-Programmer) [inline lib/core_types/bool.dart:6]: Use package imports!

**Review by Andy-Python-Programmer — APPROVED** [review]

**Review by Calamity210 — APPROVED** [review]
LGTM, for future reference is your code editor set to format on save? Our styling guide doesnt follow dart format so formatting wouldnt rly help. Ill let it by this time as I havent written a styling guide in the docs yet 👍

**Comment by Andy-Python-Programmer** [comment]
Thanks

**Pull request merged**